spec/loadbalancer/README.md

```diff
@@ -1,14 +1,17 @@
-## EC Load Balancer Spec
+![Docker Cloud Build Status](https://img.shields.io/docker/cloud/build/enterpriseconnect/loadbalancer)
+## EC Gateway Load Balancer Spec
+### Goal
+High availability for EC gateway. Load balancing network traffic across gateway instances from multiple source and target agents in platform-agnostic way.
+
+### Runtime Requirement
+- VM's with same network group/ subnet. One VM for load balancer and rest for watchers
+- OS: Linux, Windows, Darwin, Arm, etc.
+- [docker 1x.xx+](https://docs.docker.com/get-docker/)   
```

Comment:
Please consider remove-
> VM's with same network group/ subnet. One VM for load balancer and rest for watchers

```docker 1x.xx+ ``` is not a valid version. Please verify in official docker versions.

Reply:
Updated the version